config.mak.uname

```diff
@@ -556,11 +556,19 @@ else
 	ifeq ($(shell expr "$(uname_R)" : '2\.'),2)
 		# MSys2
 		prefix = /usr/
+		# Enable DEP
+		BASIC_LDFLAGS += -Wl,--nxcompat
+		# Enable ASLR (unless debugging)
+		ifneq (,$(findstring -O,$(CFLAGS)))
```

Comment:
Should this be enabled only when optimization is not `-O0`?

Reply:
The idea is to enable this only for optimized builds `-O0`, `-O2` or any other `-O<n>` option